src/repositories/live.repository.ts

```diff
@@ -0,0 +1,330 @@
+import { Repository } from '../core/repository';
+import Chance = require('chance');
+import { LiveSwitchCommentsResponseRootObject } from '../responses/live.switch-comments.response';
+import { LiveCreateBroadcastResponseRootObject } from '../responses/live.create-broadcast.response';
+import { LiveStartBroadcastResponseRootObject } from '../responses/live.start-broadcast.response';
+import { LiveEndBroadcastResponseRootObject } from '../responses/live.end-broadcast.response';
+import { LiveCommentsResponseRootObject } from '../responses/live.comments.response';
+import { LiveHeartbeatViewerCountResponseRootObject } from '../responses/live.heartbeat-viewer-count.response';
+import { LiveInfoResponseRootObject } from '../responses/live.info.response';
+import { LiveFinalViewersResponseRootObject } from '../responses/live.final-viewers.response';
+import { LiveObsSettings } from '../types/live.obs-settings';
+import { LiveViewerListResponseRootObject } from '../responses/live.viewer-list.response';
+import { LiveShowQuestionResponseRootObject } from '../responses/live.show-question.response';
+import { LiveHideQuestionResponseRootObject } from '../responses/live.hide-question.response';
+import { LiveGetQuestionsResponseRootObject } from '../responses/live.get-questions.response';
+import { LiveWaveResponseRootObject } from '../responses/live.wave.response';
+import { LiveLikeResponseRootObject } from '../responses/live.like.response';
+import { LiveLikeCountResponseRootObject } from '../responses/live.like-count.response';
+import { LiveJoinRequestCountsResponseRootObject } from '../responses/live.join-request-counts.response';
+
+export class LiveRepository extends Repository {
+
+  public async disableComments(broadcastId: string): Promise<LiveSwitchCommentsResponseRootObject> {
```

Comment:
Please, rename all of the methods in accordance with the endpoints, this is naming convention for repositories.
This method should be named `muteComment`.